docs/_includes/ds6/button.html

```diff
@@ -210,12 +210,26 @@ <h3 id="button-expand">Expand Button</h3>
                 </span>
             </button>
         </div>
+        <div class="demo__inner">
+            <button aria-expanded="false" class="expand-btn expand-btn-example">
+                <span class="expand-btn__cell">
+                    <span></span>
```

Comment:
Does this empty span need to stay there?

Reply:
It's not required, no.